apps/src/lab2/views/components/Instructions.tsx

```diff
@@ -14,7 +12,7 @@ import {
 import EnhancedSafeMarkdown from '@cdo/apps/templates/EnhancedSafeMarkdown';
 import {useAppDispatch, useAppSelector} from '@cdo/apps/util/reduxHooks';
 
-import {LabState} from '../../lab2Redux';
+import {continueOrFinishLesson} from '../../progress/progressThunks';
```

Comment:
super nit: when there's `..` in the path I like to use the fully-qualified path instead (`@cdo/apps/...`)

Reply:
Good call, done